utils/ansible/ftest/file/enable-coredumps.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# set -x
+set -u -e -o pipefail
+
+if [[ $(id -u) -ne 0 ]] ; then
+	echo "[ERROR] Could only be used by root"
+fi
+
+sed -i -E -e "/DefaultLimitCORE/s/.*/DefaultLimitCORE=infinity:infinity/" /etc/systemd/system.conf
+
+cat > /etc/security/limits.d/daos-coredumps.conf << EOF
+# /etc/security/limits.d/daos-coredumps.conf
```

Comment:
Is this just a commented out line that you can remove? It's the same path as the line above.

Reply:
> Is this just a commented out line that you can remove? It's the same path as the line above.

Not sure to understand. Line 13 is the header of the output file created at line 12 with the cat command.  From what know, it is a common practice to have the first line of such configuration file defining the location of the file.  Does it make sense ?

Reply:
ah okay. Thanks for the explanation.